.projenrc.js

```diff
@@ -1,17 +1,13 @@
 /* eslint @typescript-eslint/no-var-requires: "off" */
-const {
-  AwsCdkConstructLibrary,
-  ProjectType,
-  NodePackageManager
-} = require("projen");
+const { AwsCdkConstructLibrary, ProjectType, NodePackageManager } = require("projen");
 
 const project = new AwsCdkConstructLibrary({
   name: "datadog-cdk-constructs",
   description: "CDK Construct Library to automatically instrument Python and Node Lambda functions with Datadog",
   author: "Datadog",
   authorOrganization: true,
   entrypoint: "lib/index.js",
-  repositoryUrl: "git@github.com:DataDog/datadog-cdk-constructs.git",
+  repositoryUrl: "https://github.com/DataDog/datadog-cdk-constructs",
```

Comment:
twine was complaining that it was not a valid URL so I set it to the actual https link